README.md

```diff
@@ -26,5 +26,5 @@ Any questions let us know!
 
 <!-- DO NOT EDIT -->
```

Comment:
Anything in these comments `<!--START_SECTION:data-section-->` `<!--START_SECTION:data-section-->`  will get overridden by our [GitHub Action](https://github.com/EddieJaoudeCommunity/gh-action-community). I recommend that you revert these changes as any changes not expected will be removed.

Reply:
I made sure to only modify the image URL in data.json and README.md, everything else should be unmodified.